addons/account/models/account_move.py

```diff
@@ -361,7 +361,7 @@ def _get_counterpart(self):
             counterpart = list(counterpart)[0:2] + ["..."]
         self.counterpart = ",".join(counterpart)
 
-    name = fields.Char(required=True, string="Label")
+    name = fields.Char(string="Label")
```

Comment:
I have another use case where the required is necessary.

Let's say you automate some creation of account move lines (very frequent development process) then the developer will not left the field empty because the required element, and will force your developer to think in a proper name for the record.

i.e.: The text generated when changed the cost says "Standard price changed" there is any other reference on the record, with that it saves my a huge amount of time to audit those elements.

Accounting software is about both "Trace data and Load Data" You can not have only one of theme Journal Entry by journal entry you must have them all from an usability point of view AND technical Pov.

And be care, I need the element line by line, not by account move.